src/modules/usage/usage.repository.ts

```diff
@@ -65,6 +65,105 @@ export class SequelizeUsageRepository {
     return result.length > 0 ? this.toDomain(result[0]) : null;
   }
 
+  async getUserUsage(userUuid: string) {
+    const query = `
+        WITH yearly_sums AS (
+          SELECT
+              date_trunc('year', period) AS year,
+              SUM(delta) AS total_delta
+          FROM
+              public.usages
+          WHERE
+              type = 'yearly'
+              AND user_id = :userUuid
+          GROUP BY
```

Comment:
Why are you grouping by year if it is a yearly record? The record itself is an aggregation. Am I missing something?